lib/process-services-cloud/src/lib/task/task-filters/components/edit-task-filter-cloud.component.html

```diff
@@ -60,12 +60,17 @@
                         </div>
                     </mat-form-field>
                     <div fxFlex="23%" class="adf-edit-task-filter-checkbox" *ngIf="isCheckBoxType(taskFilterProperty)">
-                    <mat-checkbox
-                        color="primary"
-                        [formControlName]="taskFilterProperty.key"
-                        [attr.data-automation-id]="taskFilterProperty.key">
-                        {{taskFilterProperty.label | translate}}
-                    </mat-checkbox>
+                        <mat-checkbox
+                            color="primary"
+                            [formControlName]="taskFilterProperty.key"
+                            [attr.data-automation-id]="taskFilterProperty.key">
+                            {{taskFilterProperty.label | translate}}
+                        </mat-checkbox>
+                    </div>
+
+                    <div fxFlex="23%" class="adf-edit-task-filter-dateRange" *ngIf="isDateRangeType(taskFilterProperty)">
```

Comment:
same as above, the div has already a `class` assigned, you can put the flex stuff there.